.github/workflows/build_and_push.yml

```
@@ -63,11 +117,7 @@ jobs:
 
       - name: Set CI Tag
         run: |
-          BRANCH="${GITHUB_REF##*/}"
-          CI_TAG=${BRANCH#v}-ci
-          if [ ${BRANCH} = "develop" ]; then
-            CI_TAG="ci"
-          fi
+          CI_TAG := $(shell awk -F': ' '/^version:/ {print $$2}' deploy/helm/charts/Chart.yaml)
```

Syntax error for the shell script. Makefile's lazy assignment operator wouldn't work here.
```suggestion
          CI_TAG=$(shell awk -F': ' '/^version:/ {print $$2}' deploy/helm/charts/Chart.yaml)
```